src/canvas/canvasUtils.js

```diff
@@ -0,0 +1,273 @@
+// @flow
```

Comment:
I recommend using `git mv` to move a file  and commit it, since now I can't see if the file has been changed or just moved. This way we also lose the version history.

No need to change anything now, only for future reference.

Reply:
I created this file in this PR but I'll keep that in mind, thanks!